build.sbt

```diff
@@ -111,10 +111,26 @@ TaskKey[Unit]("verifyCodeFmt") := {
   }
 }
 
+val scalaReleaseSeparateSource: Def.SettingsDefinition = Compile / unmanagedSourceDirectories ++= {
```

Comment:
Why is this necessary, this functionality is automatically provided by sbt, i.e. sbt will automatically handle `scala-2`/`scala-3` folders in the way this code seems to be describing.

You only need custom handling if you want to do something like `scala-2.13+` like in pekko core

Reply:
I wasn't aware that scala-2, scala-3 worked automatically, I've removed explicit setting now.